packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/models/errors/no_mail_app_error_model.dart

```diff
@@ -0,0 +1,6 @@
+part of 'error_model.dart';
+
+class NoMailAppErrorModel extends ErrorModel {
+  @override
+  String toString() => 'NoMailAppModel.';
+}
```

Comment:
Should this be generated always or only under specific conditions?

Reply:
Moved to be part of the `feature_onboarding` brick